providers/azure/resources/azure.lr

```diff
@@ -48,7 +48,7 @@ azure.subscription @defaults ("name") {
   mariaDb() azure.subscription.mariaDbService
   // Cosmos DB resources in the subscription
   cosmosDb() azure.subscription.cosmosDbService
-  // Key vault resources in the subscription
+  // Vault resources in the subscription
```

Comment:
Isn't this a reference to the product name not the objects

Reply:
```suggestion
  // Azure Key Vault resources in the subscription
```